agora/Agora/Effect/TreasuryWithdrawal.hs

```diff
@@ -122,6 +122,10 @@ treasuryWithdrawalValidator currSymbol = makeEffect currSymbol $
         treasuryInputValuesSum = sumValues #$ ofTreasury # inputValues
         treasuryOutputValuesSum = sumValues #$ ofTreasury # outputValues
         receiverValuesSum = sumValues # datum.receivers
+        isCollateral = plam $ \cred -> P.do
```

Comment:
I would rename this to `isPubKey`, since that's what it does. I don't think there's actually a way to check from within a validator whether something is a collateral input or not :thinking:.